biometrics/extract.py

```diff
@@ -51,6 +51,12 @@ def _parse_bed_file(self):
             return
 
         self.regions = pd.read_csv(self.bed, sep='\t', header=None)
+
+        # only keep Y chrom regions
+        self.regions = self.regions[self.regions[0].isin(['Y', 'chrY'])]
+        if len(self.regions) == 0:
+            print('There are not Y chromosome regions!')
+
```

Comment:
Is this only used for the sex mismatch calculation, or why wouldn't you want to keep all regions of the bed file?

Reply:
Yes, it is only used for sex mismatch determination and only needs the Y chromosome regions. i just want it so that the person using biometrics does not have to preprocess the bed file on their end.